dev.linwood.butterfly.appdata.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>dev.linwood.butterfly</id>
+	<name>Linwood Butterfly</name>
+	<developer_name>Linwood</developer_name>
+	<summary>Powerful, minimalistic, cross-platform, opensource note-taking app</summary>
+	<metadata_license>CC0-1.0</metadata_license>
+	<project_license>GPL-3.0-only</project_license>
+	<supports>
+	  <control>pointing</control>
+	  <control>keyboard</control>
+	  <control>touch</control>
+	</supports>
+	<recommends>
+		<control>pointing</control>
+		<control>keyboard</control>
+		<control>touch</control>
+	</recommends>
+	<developer_name>LinwoodCloud</developer_name>
+	<url type="homepage">https://github.com/LinwoodCloud/butterfly</url>
```

Comment:
It looks like your homepage is https://docs.butterfly.linwood.dev/ and not the repo. I suggest the following:
```xml
<url type="homepage">https://docs.butterfly.linwood.dev/</url>
<url type="bugtracker">https://github.com/LinwoodCloud/Butterfly/issues</url>
<url type="help">https://docs.butterfly.linwood.dev/docs/1.1/intro</url>
<url type="translate">https://translate.linwood.dev/butterfly</url>
<url type="vcs-browser">https://github.com/LinwoodCloud/butterfly</url>
<url type="contributer">https://github.com/LinwoodCloud/Butterfly/blob/develop/CONTRIBUTING.md</url>
```
See the [specification](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-url) for a list of all URL types.

Reply:
Are you sure it's correct? 
dev.linwood.butterfly:             AppData problem: tag-invalid : <url> type invalid [unknown]

Error loading AppData file: AppData file /app/share/metainfo/dev.linwood.butterfly.appdata.xml was not valid

Reply:
I added this:
```xml
<url type="homepage">https://docs.butterfly.linwood.dev/</url>
	<url type="bugtracker">https://github.com/LinwoodCloud/Butterfly/issues</url>
	<url type="help">https://docs.butterfly.linwood.dev/docs/latest/intro</url>
	<url type="translate">https://translate.linwood.dev/butterfly</url>
	<url type="vcs-browser">https://github.com/LinwoodCloud/Butterfly.git</url>
	<url type="contribute">https://github.com/LinwoodCloud/Butterfly/blob/develop/CONTRIBUTING.md</url>
```

Reply:
Try to use org.flatpak.Builder instead of flatpak-builder.

First install it:
```
flatpak install org.flatpak.Builder
```

Then run it:
```
flatpak run org.flatpak.Builder #same args as flatpak-builder
```

It is Flatpak builder packaged as Flatpak. If you run flatpak-builder from your Host, it will use some things from your Hostsystem which may result in a different Build than on Flathub. With org.flatpak.Builder you get the same experience than on the Flathub Buildbot.

Reply:
Try removing the --install-deps-from=flathub argument. This doesn't seem t work inside a Flatpak.

Reply:
```
FB: Running 'flatpak info --arch=x86_64 --show-commit org.freedesktop.Sdk 21.08' on host
Failed to init: Unable to find sdk org.freedesktop.Sdk version 21.08
```

I ran
```
flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo --user
  flatpak install flathub org.freedesktop.Platform//21.08 org.freedesktop.Sdk//21.08 --user -y
  flatpak install org.flatpak.Builder --user -y
flatpak run org.flatpak.Builder --disable-rofiles-fuse --force-clean --arch=x86_64 --ccache flatpak_app flatpak.json -y -v --user
```

Reply:
I have built it using github actions and the ubuntu-20.04 version. I will look what I have done wrong

Reply:
Try running it on your normal PC. From what I have tested, the program works fine.